### ISSUE TITLE
CLI build broken

### ISSUE THREAD

**Issue by Henny022** [minishmaker/randomizer#52]
the CLI build is broken on `master`
```
/home/henny/git/TMCR/MinishCapRandomizerCLI/Program.cs(291,27): error CS0029: Cannot implicitly convert type 'RandomizerCore.Controllers.Models.ShufflerControllerResult' to 'bool' [/home/henny/git/TMCR/MinishCapRandomizerCLI/MinishCapRandomizerCLI.csproj]
/home/henny/git/TMCR/MinishCapRandomizerCLI/Program.cs(359,22): error CS0019: Operator '&&' cannot be applied to operands of type 'bool' and 'ShufflerControllerResult' [/home/henny/git/TMCR/MinishCapRandomizerCLI/MinishCapRandomizerCLI.csproj]
/home/henny/git/TMCR/MinishCapRandomizerCLI/Program.cs(376,22): error CS0019: Operator '&&' cannot be applied to operands of type 'bool' and 'ShufflerControllerResult' [/home/henny/git/TMCR/MinishCapRandomizerCLI/MinishCapRandomizerCLI.csproj]
```

**Comment by Catobat** [comment]
Will be fixed in revision 3.
https://github.com/Catobat/randomizer/commit/2761b13cf1e6d37af1792b2e0ce4dc64780fc71f

**Comment by Myth197** [comment]
Resolved by #56
